src/entities/note.cpp

```diff
@@ -1272,46 +1259,41 @@ bool Note::storeNoteTextFileToDisk() {
     file.flush();
     file.close();
 
-    this->hasDirtyData = false;
-    this->fileLastModified = QDateTime::currentDateTime();
+    this->_hasDirtyData = false;
+    this->_fileLastModified = QDateTime::currentDateTime();
 
     if (!fileExists) {
-        this->fileCreated = this->fileLastModified;
+        this->_fileCreated = this->_fileLastModified;
     }
 
     const bool noteStored = this->store();
-
-    // if note was stored but the note file wasn't renamed do some more checks
-    // whether we need to remove the old note file
-    if (noteStored && !noteFileWasRenamed) {
-        QFile oldFile(oldNoteFilePath);
-        const QFileInfo oldFileInfo(oldFile);
-        const QFile newFile(fullNoteFilePath());
-        const QFileInfo newFileInfo(newFile);
-
-        // in the end we want to remove the old note file if note was stored and
-        // filename has changed
-        // #1190: we also need to check if the files are the same even if the
-        //  name is not the same for NTFS
-        if ((fullNoteFilePath() != oldNoteFilePath) &&
-            (oldFileInfo != newFileInfo)) {
-            // remove the old note file
-            if (oldFile.exists() && oldFileInfo.isFile() &&
-                oldFileInfo.isReadable() && oldFile.remove()) {
+    QFile oldFile(oldNoteFilePath);
+    const QFileInfo oldFileInfo(oldFile);
+    const QFile newFile(fullNoteFilePath());
+    const QFileInfo newFileInfo(newFile);
+
+    // in the end we want to remove the old note file if note was stored and
+    // filename has changed
+    // #1190: we also need to check if the files are the same even if the name
+    // is not the same for NTFS
+    if (noteStored && (fullNoteFilePath() != oldNoteFilePath) &&
```

Comment:
Does those changes still do the same? :grimacing:

Reply:
Please don't slip in such big rewrites with minor variable changes. These should better go into a different pull requests where you explain what you did. :sweat_smile: the same goes for the `copyToPath` rewrite, that caused breaking of coping and moving of notes (#1764)

Reply:
I dont remember making these changes. Maybe this happened because of the renaming

Reply:
Large chunks of code were rewritten or restructured (not only that one line).

Reply:
Its best that i restore these